hs_core/hydroshare/hs_bagit.py

```diff
@@ -128,20 +119,32 @@ def create_bag(resource):
 
     # change the namespace for the 'creator' element from 'dcterms' to 'dc'
     xml_string = remdoc.data.replace('dcterms:creator', 'dc:creator')
-    with open(bagit_path + '/resourcemap.xml', 'w') as out:
+
+    # create resourcemap.xml and upload it to iRODS
+    from_file_name = '{path}/resourcemap.xml'.format(path=bagit_path)
+    with open(from_file_name, 'w') as out:
         out.write(xml_string)
+    to_file_name = '{res_id}/data/resourcemap.xml'.format(res_id=resource.short_id)
+    istorage.saveFile(from_file_name, to_file_name, False)
+
+    # call iRODS bagit rule here
+    irods_dest_prefix = "/" + settings.IRODS_ZONE + "/home/" + settings.IRODS_USERNAME
+    irods_bagit_input_path = os.path.join(irods_dest_prefix, resource.short_id)
+    bagit_input = "*BAGITDATA='{path}'".format(path=irods_bagit_input_path)
+    bagit_rule_file = getattr(settings, 'IRODS_BAGIT_RULE', 'hydroshare/irods/ruleGenerateBagIt_HS.r')
```

Comment:
You don't need a default here. Just use `settings.IRODS_BAGIT_RULE` because we control the settings and guarantee the setting exists.